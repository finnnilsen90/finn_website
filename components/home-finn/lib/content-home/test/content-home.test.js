const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer')
let { shallow, mount, render } = require('enzyme');

const Content = require('../content-home.js');

let newContent = new Content();

describe('component connection', () => {

    it('should be defined', () => {
        expect(Content).toBeDefined();
    })

})  
    
describe('<Content />', () => {

    const wrapper = render(<Content />);

    it('renders components', () => {
        expect(wrapper).toMatchSnapshot();
    });


})