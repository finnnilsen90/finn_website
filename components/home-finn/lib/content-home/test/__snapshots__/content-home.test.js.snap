// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> renders components 1`] = `
<div
  class="content-home_Container"
>
  <img
    class="profile"
    src="test-file-stub"
  />
  <p
    class="overview-finn"
  >
    My name is Finn Nilsen. I have been working in the tech industry for 5+ years in a variety of roles. Iâ€™ve been a software developer, digital marketer, and product manager. I have a passion for building great software products and leading technical teams. This website is an overview of my career so far and how I can help any company I work with grow.
  </p>
</div>
`;
